controller/WebController.php

```diff
@@ -559,22 +559,54 @@ public function invokeGenericErrorPage($request, $message = null)
      */
     public function invokeChangeList($request, $prop='dc:created')
     {
-        // set language parameters for gettext
-        $this->setLanguageProperties($request->getLang());
-        $vocab = $request->getVocab();
-        $offset = ($request->getQueryParam('offset') && is_numeric($request->getQueryParam('offset')) && $request->getQueryParam('offset') >= 0) ? $request->getQueryParam('offset') : 0;
-        $changeList = $vocab->getChangeList($prop, $request->getContentLang(), $request->getLang(), $offset);
+        $bydate = $this->formatChangeList($request, $prop);
```

Comment:
I think it would be cleaner to call getChangeList here in invokeChangeList, instead of within formatChangeList.
So something like

```php
$changeList = $this->getChangeList($request, $prop); // extract $offset and $limit from $request within getChangeList
$bydate = $this->formatChangeList($changeList, $request->getLang());
```

This would also mean less data being passed around as parameters.